assets/InstallPSCorePolicyDefinitions.ps1

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+    Group Policy tools use administrative template files (.admx, .adml) to populate policy settings in the user interface.
+    This allows administrators to manage registry-based policy settings.
+    This script installes PowerShell Core Administrative Templates for Windows.
+.Notes
+    The PowerShellCoreExecutionPolicy.admx and PowerShellCoreExecutionPolicy.adml files are
+    expected to be at the location specified by the Path parameter with default value of the location of this script.
+#>
+[CmdletBinding()]
+param
+(
+    [ValidateNotNullOrEmpty()]
+    [string] $Path = $PSScriptRoot
+)
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+function Test-Elevated
+{
+    [CmdletBinding()]
+    [OutputType([bool])]
+    Param()
+
+    # if the current Powershell session was called with administrator privileges,
+    # the Administrator Group's well-known SID will show up in the Groups for the current identity.
+    # Note that the SID won't show up unless the process is elevated.
+    return (([Security.Principal.WindowsIdentity]::GetCurrent()).Groups -contains "S-1-5-32-544")
```

Comment:
This won't work on nano server.  So we should add another check for nano server.

Reply:
Added check.